commands-yml/commands/interactions/touch/multi-touch-perform.yml

```diff
@@ -0,0 +1,107 @@
+---
+name: Touch Perform
+short_description: Perform a multi touch action sequence
+   
+
+example_usage:
+  java:
+    |
+      MultiTouchAction action = new MultiTouchAction(driver);
+      action.down(10, 10);
```

Comment:
There is no down|moveTo|etc methods in `MultiTouchAction`. Should be something like [this](https://github.com/appium/java-client/blob/master/docs/Touch-actions.md#how-to-construct-multiple-touch-action)